pages/settings/profile.vue

```diff
@@ -75,9 +120,9 @@
 </script>
 
 <template>
-	<Modal v-model="avatarCropperIsOpen">
+	<Modal v-model="avatarCropperIsOpen" title="更新头像">
 		<div style="width: 20vw; height: 20vw;">
```

Comment:
```suggestion
 		<div :style="{ width: '20dvw', height: '20dvw' }">
```

回去我找个 ESLint 规则把这种行内样式写法给毙了。

理论上现在改动的方法也不推荐，应该添加一个 class，然后在 `<style>` 里面编样式。

Reply:
Fixed
不知道这么写好不好
``` html
<div class="avatar-cropper">
```
``` css
.avatar-cropper {
	--avatar-cropper-side-length: 350px;
	width: var(--avatar-cropper-side-length);
	height: var(--avatar-cropper-side-length);

	@media (width <= 450px) {
		--avatar-cropper-side-length: 80dvw; /// 对于图片切割器，不建议使用响应式，因为切割器内部被切割的图片不会随之改变尺寸，但考虑到极端小尺寸的适配问题，且只有极少数场景会改变浏览器宽度
	}
}
```